.editorconifg

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
```

Comment:
IMO we should wait on switching to LF until we batch-change every file (if we decide to), otherwise small changes to any file will include line endings in diffs

Reply:
Oh I just saw the gitattributes change, if that solves this then nevermind me

Reply:
Yeah I think this can probably be removed in favor of what is in .gitattributes yes?

Reply:
New comment below to remove the line endings enforcement on local copy.